.gitignore

```diff
@@ -6,3 +6,4 @@ artifacts
 node.json
 .idea
 cairo/
+target/
```

Comment:
why is this needed here?

Reply:
for some reason running it generated a target dir that it useless so added it to my gitignore